decidim/decidim
# Strip the `<p>` tags from inside the heading elements

#### :tophat: What? Why?
The organization description is usually wrapped inside a `<p>` element which causes a HTML validation error.

Strip out the surrounding `<p>` tags in the sub hero element.

#### :pushpin: Related Issues
- Related to #5942, #6246, #6124, #5684

#### Testing
1. Go to https://validator.w3.org/nu/#textarea
2. Add the following content there:
```
<!doctype html>
<html lang="en">
<head>
  <title>Title</title>
</head>
<body>
  <h1><p>Don't use p tag inside headings</p></h1>
</body>
</html>
```
3. See the error

#### :clipboard: Checklist
